.travis.yml

```diff
@@ -15,11 +14,11 @@ matrix:
             apt:
               packages: valgrind
           compiler: gcc
-          env: BUILD_CONFIGURATION=0 WITH_COVERAGE=1
+          env: BUILD_CONFIGURATION=0 WITH_COVERAGE=1 BUILD_DIR="/tmp/cppmicroservices_builds"
```

Comment:
Yes, the space is causing the final upload script to trip. I had to reconfirmed by running some builds in CI. So I made the environment variable specific to code coverage builds. We cover the  use case with a space in the directory path, in the builds without code coverage.